dask_sql/physical/rex/core/call.py

```diff
@@ -600,6 +605,25 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        if df.dtype == "object":
+            df = pdlike.to_datetime(df)
+            return df.strftime(format)
+        else:
+            df = pdlike.to_datetime(df, unit="s")
+            df = df.strftime(format)
+            result = [timestamp for timestamp in df]
+            return pdlike.Series(result)
```

Comment:
```suggestion
            df = pdlike.to_datetime(df, unit="s")
            df = df.strftime(format)
            result = np.array([])
            for timestamp in df:
                result = np.append(result, timestamp)
            return pdlike.Series(result)
```

Hi @ayushdg would this be a more preferred way?

Reply:
I think I understand, is it the case that `df.strftime` returns a string or list of strings that we need to reconvert into a series?

Reply:
Yes, but when I try to append directly to an empty series I got a `TypeError: cannot concatenate object of type '<class 'str'>'; only Series and DataFrame objs are valid`.